src/CommunityToolkit.Maui.Core/Essentials/SpeechToText/SpeechToTextImplementation.windows.cs

```diff
@@ -103,7 +100,7 @@ async Task StopRecording(CancellationToken cancellationToken)
 				speechRecognizer.ContinuousRecognitionSession.Completed -= OnCompleted;
 
 				cancellationToken.ThrowIfCancellationRequested();
-				await speechRecognizer.ContinuousRecognitionSession.StopAsync();
+				await speechRecognizer.ContinuousRecognitionSession.StopAsync().AsTask(cancellationToken);
 			}
```

Comment:
What is the benefit of it?

Reply:
[`StopAsync()` returns an `IAsyncAction`](https://learn.microsoft.com/en-us/uwp/api/windows.media.speechrecognition.speechcontinuousrecognitionsession.stopasync?view=winrt-22621) (not `Task`) and does not accept a `CancellationToken` as a parameter. 

The only way for us to associate our `CancellationToken` with this long-running asynchronous method, is to first convert it to a `Task` using `.AsTask(CancellationToken)`. 

If it were a `Task`, I could just use `.WaitAsync(cancellationToken)`

Reply:
WinUI has a few of these weird APIs, like [`IAsyncAction`](https://learn.microsoft.com/en-us/uwp/api/windows.media.speechrecognition.speechcontinuousrecognitionsession.stopasync?view=winrt-22621), that are awaitable, but are not derived from `Task`